_licenses/agpl-3.0.txt

```diff
@@ -4,7 +4,7 @@ nickname: GNU AGPLv3
 redirect_from: /licenses/agpl/
 source: http://www.gnu.org/licenses/agpl-3.0.txt
 
-description: The GNU GPL family of licenses is the most widely used free software license and has a strong copyleft requirement. When distributing derived works, the source code of the work must be made available under the same license. GNU AGPLv3 is distinguished from GNU GPLv3 in that hosted services using the code are considered distribution and trigger the copyleft requirements.
+description: Permissions of this strongest copyleft license are conditioned on making available complete source code of licensed works and modifications, which include larger works using a licensed work, under the same license when distributed. Copyright and license notices must be preserved. Contributors provide an express grant of patent rights to users. Additionally, when a modified version is used to provide a service over a network, the complete source code of the modified version must be made available to users of the service.
```

Comment:
By all means, I'm not suggesting we should assume our reader is actually five (or write as if they were), but likewise, we shouldn't assume that the reader absolutely needs an accurate and complete summary, or that they have the patience/stomach to grok through legalease. 

We should be comfortable being a little wrong if it means making things easier for the vast majority of users. Our job is to absorb at least some of that complexity on behalf of the reader and find a happy medium (where on that continuum we land is obviously up to you, but I think we can both agree that both 100% accurate/complete and 100% readable - the two extremes - are not good goals to strive for).

> as those details matter

They do for the edge cases, but do they for 50% of users? [80%?](http://ben.balter.com/2016/03/08/optimizing-for-power-users-and-edge-cases/) 90%?